compiler/src/codegen/compcore.re

```diff
@@ -3621,10 +3596,10 @@ let compile_wasm_module = (~env=?, ~name=?, prog) => {
 
   let compile_all = () => {
     ignore @@ compile_functions(wasm_mod, env, prog);
+    ignore @@ compile_tables(wasm_mod, env, prog);
```

Comment:
Why does the order matter here?

Reply:
Compiling imports needs to happen after anything that imports something is done so it knows what imports need to get included (and tables import an offset).